TestCases/hybrid_regression.py

```diff
@@ -134,6 +134,14 @@ def main():
     poiseuille_profile.test_vals = [-12.494752, -7.712204, -0.000000, 2.085796]
     test_list.append(poiseuille_profile)
 
+    # 2D Periodic
```

Comment:
```suggestion
    # 2D rotational Periodic
```
a small additional hint